packages/flutter/lib/src/rendering/shifted_box.dart

```diff
@@ -511,6 +511,20 @@ class RenderPositionedBox extends RenderAligningShiftedBox {
   }
 }
 
+/// How much space should be occupied by the [OverflowBox] if there is no
+/// overflow.
+enum OverflowBoxFit {
+  /// The widget will size itself to be as large as the parent allows.
+  max,
+
+  /// The widget will follow the child's size.
+  ///
+  /// More specifically, the render object will size itself to match the size of
+  /// its child within the constraints of its parent or be as small as the
```

Comment:
Super minor optional clarification/simplification:
```suggestion
  /// its child within the constraints of its parent, or as small as the
```